### PR TITLE
Remove two greys

### PR DESCRIPTION
**Pull request by sammorrisdesign-zz**
That's right! :bomb: :bomb: :bomb:

This PR removes two of our greys from the project. Most of this PR is find and replace.

You can see the reduced palette on the [pasteup-palette demo page](http://guardian.github.io/pasteup-palette/demo/)
## What does this change?

We now have two less colours in the palette and now only six neutral greys across the site.
## Screenshots

The difference between the greys I've removed and the ones I've used in their place aren't distinguishable to the human eye.

(cc @Pearson82 @superfrank @zeftilldeath)
## 

_Does this PR meet the [contributing guidelines](https://github.com/guardian/frontend/blob/issue_pr_templates/.github/CONTRIBUTING.md#submission)?_

### CODE REVIEW

File: static/src/stylesheets/components/pasteup-forms/src/_forms.scss [hunk 1]
```diff
@@ -11,34 +11,70 @@ category: Common
 <form class="form">
```
Comment (desbo) [inline static/src/stylesheets/components/pasteup-forms/src/_forms.scss]: well this was confusing

**Comment by zeftilldeath** [comment]
:+1: My favourite pull request of yours so far.

Ps. They are distinguishable to my human :eyes: and I can't wait to be subtly unnerved by the website after the changes go live.

File: static/src/stylesheets/components/pasteup-forms/src/_forms.scss [hunk 2]
```diff
@@ -60,246 +96,276 @@ category: Common
...
+
 .fieldset {
     border: 0;
-    border-top: $pasteup-forms-fieldset-border-top;
-    display: table;
-    padding: (($gs-baseline/3)*4 - 1) 0 ($gs-baseline*2);
+    padding: 0;
     margin: 0;
+    border-bottom: $pasteup-forms-fieldset-border-bottom;
+    display: table;
```
Comment (desbo) [inline static/src/stylesheets/components/pasteup-forms/src/_forms.scss]: there's quite a bit here that doesn't seem related to removing the greys?

**Comment by zeftilldeath** [comment]
PS my :+1: is hypothetical. God knows if you've broken the website or not.

File: static/src/stylesheets/components/pasteup-forms/src/_forms.scss (hunk 2 above)
Comment (sammorrisdesign) [inline static/src/stylesheets/components/pasteup-forms/src/_forms.scss]: Yeah it turns out that people have been updating the components directly in the `frontend` repo rather than doing releases. So me updating them properly seems to have gotten a whole bunch of updates.

**Comment by superfrank** [comment]
My least favourite of your pull requests so far. I guess we'll have to keep our  peeled for if this looks odd anywhere/any of those greys were sat on top of each other as dotted rules etc.

**Comment by Pearson82** [comment]
Nice move 👍

File: static/src/stylesheets/components/pasteup-forms/src/_forms.scss (hunk 2 above)
Comment (desbo) [inline static/src/stylesheets/components/pasteup-forms/src/_forms.scss]: ah gotcha, yeah it's a quirk of having these files checked in i suppose

**Comment by sammorrisdesign** [comment]
I'm not merging this in right now as our bower components have diverged from the ones in the project. @sndrs and I have a plan...
